cpplint_unittest.py

```diff
@@ -4165,21 +4166,21 @@ def testControlClauseWithParensNewline(self, keyword):
     # The % 2 part is pseudorandom whitespace-support testing
     self.TestLintContains(
       f'{keyword}{["", " "][len(keyword) % 2]}(condition)'
-        f'{[" ", ""][len(keyword) % 2]}[[unlikely]]'
-        f'{[" ", ""][len(keyword) % 2]}{{'
-        f'{["", " "][len(keyword) % 2]}do_something(); }}',
+      f'{[" ", ""][len(keyword) % 2]}[[unlikely]]'
+      f'{[" ", ""][len(keyword) % 2]}{{'
+      f'{["", " "][len(keyword) % 2]}do_something(); }}',
       f'Controlled statements inside brackets of {keyword} clause'
-        f' should be on a separate line  [whitespace/newline] [5]'
+      f' should be on a separate line  [whitespace/newline] [5]'
```

Comment:
I don't like these string-break formatting changes from what I presume is pyproject-fmt... They just make the long strings a bit less readable.

Reply:
These changes are required by `flake8` which lints Python code.  Let’s not waste code review time and attention things that `ruff format` can standardize across the codebase.

`pyproject-fmt` formats only the `pyproject.toml`.

Reply:
`yapf`, `psf/black`, and `ruff format` will remove this extra indentation.